### DISCUSSION TITLE
[Onvif] 10 Second Late HSKV Recording compared to reolink local motion video

### DISCUSSION THREAD

**Discussion by vegedb** [koush/scrypted#770] [Q&A] [answered]
# Reolink Local Clip
Video Clip starts at 15:37:18
# Event
15:37:26 = motionDetected: true

# Console
listening events
[HomeKit]: HomeKit Connection 10.5.1.20
[HomeKit]: snapshot requested for reason: 1
[Rebroadcast Plugin]: Selected stream Profile000_MainStream
[Rebroadcast Plugin]: Profile000_MainStream prebuffer session started
stream url rtsp://192.168.1.82:554/
[Rebroadcast Plugin]: rtsp outgoing message
[Rebroadcast Plugin]:  OPTIONS rtsp://10.5.1.49:554/ RTSP/1.0
[Rebroadcast Plugin]: CSeq: 0
[Rebroadcast Plugin]: User-Agent: Scrypted
[Rebroadcast Plugin]: 
[Rebroadcast Plugin]: rtsp incoming message
[Rebroadcast Plugin]:  RTSP/1.0 200 OK
[Rebroadcast Plugin]: CSeq: 0
[Rebroadcast Plugin]: Date: Thu, May 04 2023 15:37:27 GMT
[Rebroadcast Plugin]: Public: OPTIONS, DESCRIBE, SETUP, TEARDOWN, PLAY, PAUSE, GET_PARAMETER, SET_PARAMETER
[Rebroadcast Plugin]: 

## Problem
All HSKV clips are recorded this way, 10 sec late when motion is detected. Hope this could be fixed.

**Comment by koush** [comment, marked as answer]
Reolink may be slow to deliver motion events.
